vtkext/private/module/vtkF3DUIActor.h

```diff
@@ -60,6 +60,12 @@ class vtkF3DUIActor : public vtkProp
    * Empty by default
    */
   void SetDropText(const std::string& info);
+
+  /**
+   * Set the dropzone binds information
+   * Each pair contains a key and its associated description
```

Comment:
lets be more precise, something like:

```suggestion
   * Each pair contains an interaction bind and its associated description
```